mirascope/base/types.py

```diff
@@ -295,3 +296,94 @@ def output_tokens(self) -> Optional[Union[int, float]]:
         If there is no output_tokens, this method must return None.
         """
         ...  # pragma: no cover
+
+
+BaseCallResponseT = TypeVar("BaseCallResponseT", bound=BaseCallResponse)
+BaseCallResponseChunkT = TypeVar("BaseCallResponseChunkT", bound=BaseCallResponseChunk)
+MessageParamT = TypeVar("MessageParamT", bound=Any)
```

Comment:
These are already defined in `mirascope/base/calls.py`. Does it make sense to consolidate all the typevars and prevent duplicates

Reply:
Sure, done.